### DISCUSSION TITLE
I need help installing this

### DISCUSSION THREAD

**Discussion by Leia286** [release-drafter/release-drafter#966] [Q&A]
I need help dealing with the installation in one of the repos I maintain 
https://github.com/Leia286/RandomCSS/tree/main/.github
Can someone help me out here? I don't really know YAML. Please help with both yaml files.

**Comment by ssbarnea** [comment]
@TimonVS @jetersen To avoid getting the project hammered  with support questions you could just enable the Discussions feature and convert fake issues into discussions. It was a god-sent for few projects I manage.
